docker-entrypoint-initdb.d/001-init.sql

```diff
@@ -0,0 +1,67 @@
+/*M!999999\- enable the sandbox mode */ 
+-- MariaDB dump 10.19  Distrib 10.11.9-MariaDB, for debian-linux-gnu (x86_64)
+--
+-- Host: localhost    Database: serlo
+-- ------------------------------------------------------
+-- Server version	10.11.9-MariaDB-ubu2204
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8mb4 */;
+/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
+/*!40103 SET TIME_ZONE='+00:00' */;
+/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+--
+-- Current Database: `serlo`
+--
+
+CREATE DATABASE /*!32312 IF NOT EXISTS*/ `serlo` /*!40100 DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci */;
+
+USE `serlo`;
+
+--
+-- Table structure for table `lti_entity`
+--
+
+DROP TABLE IF EXISTS `lti_entity`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `lti_entity` (
+  `id` bigint(20) NOT NULL AUTO_INCREMENT,
+  `resource_link_id` varchar(255) DEFAULT NULL,
+  `custom_claim_id` varchar(255) DEFAULT NULL,
+  `edusharing_node_id` varchar(255) DEFAULT NULL,
+  `content` longtext DEFAULT NULL,
+  `id_token_on_creation` text NOT NULL,
+  PRIMARY KEY (`id`),
+  KEY `idx_lti_entity_custom_claim_id` (`custom_claim_id`),
+  KEY `idx_lti_entity_edusharing_node_id` (`edusharing_node_id`)
+) ENGINE=InnoDB AUTO_INCREMENT=2 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_520_ci;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `lti_entity`
+--
+
+LOCK TABLES `lti_entity` WRITE;
+/*!40000 ALTER TABLE `lti_entity` DISABLE KEYS */;
+INSERT INTO `lti_entity` VALUES
+(1,NULL,'00000000-0000-0000-0000-000000000000',NULL,NULL,'{}');
```

Comment:
It would be great to have this also in the Uberspace development and staging environment to be able to quickly test there using Saltire. I think we could add this to `setup_uberspace.sh`

Reply:
The previous setup was so that we would have these data in all environment, also in production. I think we can keep so.

All right, I've added to the `setup_uberspace.sh `.